src/mame/drivers/gigatron.cpp

```diff
@@ -70,7 +77,27 @@ void gigatron_state::machine_reset()
 {
 }
 
+uint16_t gigatron_state::fetch_outx()
+{
+	return 0;
+}
+
+void gigatron_state::blinkenlights()
+{
+	uint16_t light = fetch_outx() & 0xF;
+	lights_changed ^= light;
+}
+
 static INPUT_PORTS_START(gigatron)
+	PORT_START("INPUTS")
```

Comment:
This definition has no relation to the kind inputs this system should actually have.

Reply:
Changed it to GAMEPAD because the system has a DB-9 Joystick port

Reply:
OK, I really was wrong about this.